### PR TITLE
Docs: configuration, environment variables, fixtures

### LINKED ISSUE #420: Document Content Security Policy environment/settings
See environment variables in [`.env.sample`](https://github.com/cal-itp/benefits/blob/dev/.devcontainer/.env.sample#L18) and Django settings in [`settings.py`](https://github.com/cal-itp/benefits/blob/dev/benefits/settings.py#L203).

Could go in a new section like `Configuration` or similar.

### LINKED ISSUE #421: Document reCAPTCHA environment/settings
See environment variables in [`.env.sample`](https://github.com/cal-itp/benefits/blob/dev/.devcontainer/.env.sample#L28) and Django settings in [`settings.py`](https://github.com/cal-itp/benefits/blob/dev/benefits/settings.py#L243).

Could go in [`Getting Started`](https://github.com/cal-itp/benefits/tree/dev/docs/getting-started) as a new page or in a new section like `Configuration` or similar.

### LINKED ISSUE #422: Document Rate Limit environment/settings
See environment variables in [`.env.sample`](https://github.com/cal-itp/benefits/blob/dev/.devcontainer/.env.sample#L33) and Django settings in [`settings.py`](https://github.com/cal-itp/benefits/blob/dev/benefits/settings.py#L232).

Could go in [`Getting Started`](https://github.com/cal-itp/benefits/tree/dev/docs/getting-started) as a new page, or in a new section like `Configuration` or similar.

### LINKED ISSUE #423: Document OAuth environment/settings
See #414 for the new environment/settings that were introduced.

Let's create a new section in the docs called `Configuration` or similar, and we can group this doc together with (eventually), not necessary for this task):

* #420 
* #421 
* #422

### PR DESCRIPTION
**Pull request by thekaveman**
Closes #420.
Closes #421.
Closes #422.
Closes #423.

Definitely room for improvement and expansion, but hopefully this sets up a good starting place.

### CODE REVIEW

**Review by machikoyasuda — DISMISSED** [review]

File: docs/configuration/oauth.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### `DJANGO_OAUTH_CLIENT_ID`
+
+This application's client ID, as registered with the OAuth/OIDC server.
+
+### `DJANGO_OAUTH_CLIENT_NAME`
+
+The internal label of the OAuth client within this application.
+
+See the [`OAUTH_CLIENT_NAME`](#oauthclientname) setting for more.
```
Comment (angela-tran) [inline docs/configuration/oauth.md:41]: Very small thing I noticed: the anchor link on the page is #oauth_client_name

Comment (thekaveman) [reply docs/configuration/oauth.md:41]: Thanks! It's weird, VS Code will help you fill in these anchor links to local docs, and it mostly gets them right... except for when they contain `_` underscores.

**Review by afeld — APPROVED** [review]
This is great work! Since there aren't any docs for these things right now, I suggest we merge sooner than later and fix/tweak things as we go, rather than nitpicking anything in this pull request.

**Review by angela-tran — DISMISSED** [review]
This is really great documentation! Thank you @thekaveman !

File: docs/configuration/oauth.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# OAuth settings
```
Comment (afeld) [inline docs/configuration/oauth.md:1]: Not a blocker, but let's include information (sooner or later) about where to get these values from.

File: docs/configuration/oauth.md (hunk 1 above)
Comment (thekaveman) [reply docs/configuration/oauth.md]: Fixed in [29a1a30](https://github.com/cal-itp/benefits/pull/426/commits/29a1a300dfc558590d6839e3d87680de7eeae9cc)

**Review by angela-tran — APPROVED** [review]

**Pull request merged**